pkg/controller/util.go

```diff
@@ -30,7 +30,8 @@ func calcScaleUpDelta(allNodes []*v1.Node, cpuPercent float64, memPercent float6
 // calcPercentUsage helper works out the percentage of cpu and mem for request/capacity
 func calcPercentUsage(cpuRequest, memRequest, cpuCapacity, memCapacity resource.Quantity) (float64, float64, error) {
 	if cpuCapacity.MilliValue() == 0 || memCapacity.MilliValue() == 0 {
-		return 0, 0, errors.New("cannot divide by zero in percent calculation")
+		// Needs to return nil for just in case if ASG size is zero.
+		return 0, 0, nil
```

Comment:
returning `0, 0` here for nodeCapacity of 0 means that if the podRequests is `> 0` then it will register as `0%` and scale down rather than scale up (because the actual usage is `> 100%` in that case)?

Perhaps had an extra argument to pass in the capacity of 1 node (from the node capacity cache idea) and use that to determine the actual percentage when the nodes is 0 and pods `> 0`? 

Also does this now need to handle nodes (capacity) of `> 0` and pods (request) of `0`? This would be a usage of 0% and need a scale down.